plugins/gradle/src/index.ts

```diff
@@ -238,8 +242,89 @@ export default class GradleReleasePluginPlugin implements IPlugin {
       ]);
     });
 
+    auto.hooks.canary.tapPromise(
+      this.name,
+      async ({ dryRun, quiet }) => {
```

Comment:
This hook receives `canaryIdentifier` that should be used to create the canary version